app/main.css

```diff
@@ -557,3 +557,29 @@ input[type="range"]:hover{
 
 /* Frame rate input */
 #input-fr-change { margin-top: 0.3em; }
+
+/* ========== NOTIFICATION BAR ============== */
+.notification {
+  width: 60vw;
+  min-height: 3em;
+  position: absolute;
+  left: 18vw;
```

Comment:
I tried to make it roughly center. I didn't do an exact measurement or anything, so this may need tweaking.